dir.common.props

```diff
@@ -65,7 +65,7 @@
 
   <PropertyGroup>
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">3.0.1</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">3.0.2</PackageVersion>
 
     <!-- Set the boolean below to true to generate packages with stabilized versions -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

Comment:
To avoid confusion, can you remove the stabilization properties from here? coreclr never stabilizes.